File: codewars/7kyu/bumps-in-the-road/bumps-in-the-road.py
"""
bumps-in-the-road
codewars/7kyu/Bumps in the Road
Difficulty: 7kyu
URL: https://www.codewars.com/kata/57ed30dde7728215300005fa/
"""

def bumps(road):
    return 'Woohoo!' if road.count('n') < 16 else 'Car Dead'


def test_bump():
    assert bumps("n") == "Woohoo!"
    assert bumps("_nnnnnnn_n__n______nn__nn_nnn") == "Car Dead"
    assert bumps("______n___n_") == "Woohoo!"
    assert bumps("nnnnnnnnnnnnnnnnnnnnn") == "Car Dead"

File: codewars/7kyu/bumps-in-the-road/bumps-in-the-road.js
// bumps-in-the-road
// codewars/7kyu/Bumps in the Road
// Difficulty: 7kyu
// URL: https://www.codewars.com/kata/57ed30dde7728215300005fa/

function bump(x) {
    // if (x.match(/n/g)?.length > 15) {
    if ((x.match(/n/g) || '').length > 15) {
      return 'Car Dead';
    }
    return 'Woohoo!';
  }
  
  test("Example tests", () => {
    expect(bump("n")).toEqual("Woohoo!");
    expect(bump("_nnnnnnn_n__n______nn__nn_nnn")).toEqual("Car Dead");
    expect(bump("______n___n_")).toEqual("Woohoo!");
  });
  